src/clients/c++/perf_client/load_manager.cc

```diff
@@ -47,6 +49,17 @@
 
 namespace {
 
+// Just keep the alpha numeric characters in the name
```

Comment:
Simply remove any slashes in the name (front and back both). Just like a directory '_' and other special characters will work fine.

Reply:
Done